src/Pages/suna/Login/Login.js

```diff
@@ -1,9 +1,85 @@
-import React from "react";
-class LoginSuna extends React.Component{
-    render(){
-        return(
-            <div></div>
-        )
-    }
+import React from 'react';
+import {Link, withRouter} from 'react-router-dom';
+import './Login.scss';
+
+class LoginSuna extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      pw: "",
+      ready: "",
+    };
+  }
+
+  goToMain = () => {
+    if(this.state.ready === true){
```

Comment:
state.ready의 불리안 값을 이용하면 아래와 같이 코드를 더 간결하게 줄일 수 있습니다. 

```suggestion
    if(this.state.ready){
```